src/userForm/UserForm.tsx

```diff
@@ -1,71 +1,94 @@
-        </h2>
+      <div className="h-screen w-screen overflow-hidden absolute -z-20">
+        <div className="w-[600px] h-[550px] bg-primary_blue absolute -z-20 rotate-45 rounded-badge left-[-18.5rem] top-[-15rem] flex-0"></div>
+        <div className="w-[300px] h-[150px] border-solid border-12 border-secondary_orange absolute rounded-badge -z-10 left-[6rem] top-[-8rem] rotate-45 flex-0"></div>
+        <div className="w-[600px] h-[550px] bg-primary_blue absolute -z-20 rotate-45 rounded-badge right-[-15rem] bottom-[-25rem] flex-0"></div>
+        <div className="w-[300px] h-[150px] border-solid border-12 border-secondary_orange absolute rounded-badge -z-10 right-[-14rem] bottom-[4rem] rotate-45 flex-0"></div>
       </div>
-      <Formik
-        initialValues={{ username: "", rememberMe: false }}
-        validate={(values) => {
-          const errors: Partial<FormikValues> = {};
-          if (!values.username) {
-            errors.username = "Required";
-          } else if (!/^[A-Za-z0-9_]{3,15}$/i.test(values.username)) {
-            errors.username = "Invalid email address";
-          }
-          return errors;
-        }}
-        onSubmit={(values, { setSubmitting }) => {
-          //when the form is submitted, the values are displayed in an alert
-          //write the values to json file if rememberMe is checked, and then that is the list of saved users that you will display on the saved menu
-          alert(JSON.stringify(values, null, 2));
-          setSubmitting(false);
-        }}
+      <div
+        className="flex flex-col bg-off_white w-4/5 h-2/5 rounded-2xl shadow-3xl px-8 py-7 justify-around overflow-hidden lg:w-3/5"
+        id="formContainer"
       >
-        {({ isSubmitting }) => (
-          <Form className="w-full flex flex-col justify-center items-start gap-5">
-            <Field
-              type="text"
-              name="username"
-              placeholder="Enter your username"
-              className="h-10 w-full border-solid border-2 border-dark_off_white rounded-md p-2 bg-transparent text-dark_text"
-            />
-            <label className="text-dark_text text-sm">
-              <Field
-                type="checkbox"
-                name="rememberMe"
-                className="mr-4 bg-dark_off_white"
+        <div className="flex flex-col gap-1.5">
+          <h1 className="text-dark_text text-2xl lg:text-3xl">Find a User</h1>
+          <h2 className="text-ligher_text text-sm lg:text-base">
+            Enter the Github username of the account you would like to view.
+          </h2>
+        </div>
+        <Formik
+          initialValues={{ username: "", rememberMe: false }}
+          validate={(values) => {
+            const errors: Partial<FormikValues> = {};
+            if (!values.username) {
+              errors.username = "Required";
+            } else if (!/^[A-Za-z0-9_]{3,15}$/i.test(values.username)) {
+              errors.username = "Invalid email address";
```

Comment:
I see that input required is a username yet the error refers to an "Invalid email address". It can be a bit confusing especially if Github allows logging through either the username or email address

Reply:
In addition to this, if you do decide to accept both the email address and the username, just remember that you would then need to add regex to test if the username is a valid email address as well.

Reply:
Another thing is to build custom `validator` functions that take in something and return a `boolean` for valid or not.